python/ray/serve/_private/utils.py

```diff
@@ -619,3 +620,27 @@ def time(self) -> float:
 class Timer(TimerBase):
     def time(self) -> float:
         return time.time()
+
+
+def get_capacity_adjusted_num_replicas(
```

Comment:
This feels like it does not belong in `utils.py`, but rather `deployment_state.py`

Reply:
This method is used in both `deployment_state.py` and `autoscaling_policy.py`. If we move it to `deployment_state.py`, then `autoscaling_policy.py` would have to import `deployment_state.py`, which seems like an awkward dependency structure. I kept it in `utils.py` because it makes more sense for `deployment_state.py` and `autoscaling_policy.py` to import from `utils.py` rather than each other.

Let me know what you think. Is there another common file that this method could be added to?